multichain-testing/README.md

```diff
@@ -16,12 +16,18 @@ The `agoric` software revision includes the vats necessary for building and test
 
 ## Initial Setup
 
-First, ensure you have Kubernetes available. See https://docs.cosmology.zone/starship/get-started/step-2.
+Install the relevant dependencies:
+
+```sh
+yarn install
+```
+
+Ensure you have Kubernetes available. See https://docs.cosmology.zone/starship/get-started/step-2.
 
 The following will install `kubectl`, `kind`, `helm`, and `yq` as needed.
 
 ```sh
-make clean setup
```

Comment:
`clean` shouldn't ever fail. If the binaries don't exist, then the desired state is already reached. Instead of removing this please make clean more robust.

Reply:
Addressed. PTAL.